rulset.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<ruleset name="Smile Lab PHPMD applied ruleset"
+         xmlns="http://pmd.sf.net/ruleset/1.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd"
+         xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">
+
+    <description>Coding standard for SmileLab projects.</description>
```

Comment:
I think we should replace "SmileLab projects" by "Magento projects" here (this standard will not work on other frameworks)